server/src/main/java/org/opensearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -1098,6 +1107,107 @@ private void asyncCleanupUnlinkedShardLevelBlobs(
         }
     }
 
+    protected boolean releaseRemoteStoreLockAndCleanup(
```

Comment:
Instead of returning boolean why can we just let the exception flow to the caller ?